src/constants/aaveConstants.ts

```diff
@@ -0,0 +1 @@
+export const AAVE_SUPPORT_ARTICLE_URL = 'https://www.edge.app/aave/%s'
```

Comment:
Let's start using the new wildcard conventions here, AAVE_SUPPORT_ARTICLE_URL_1S